vizro-core/tests/e2e/vizro/custom_components/custom_charts/table_custom.py

```diff
@@ -0,0 +1,21 @@
+from typing import Optional
+
+from dash import dash_table
+
+from vizro.models.types import capture
+
+
+@capture("table")
+def custom_table(data_frame=None, chosen_columns: Optional[list[str]] = None):
```

Comment:
Took me a bit to understand why you chose a custom table instead of just using `dash_ag_grid` directly. 
```suggestion
def table_with_filtered_columns(data_frame=None, chosen_columns: Optional[list[str]] = None):
```